change/@azure-msal-browser-2020-07-31-16-28-44-add-beachball-config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
+  "comment": "Adding beachball to the build steps",
+  "packageName": "@azure/msal-browser",
+  "email": "prithvi.kanherkar@gmail.com",
```

Comment:
Is this intentional? Should we have a team email that we can add here if really needed?

Reply:
I think this is based on the github account making the commit. I need to change my primary email on my account I think?